Jenkinsfile

```diff
@@ -198,8 +198,13 @@ def job = {
                                 sh "docker pull ${config.dockerRegistry}${dockerRepo}:${config.cp_version}-latest"
                             }
 
-                            // Set the project versions in the pom files
                             sh "set -x"
+
+                            // set the proper mavne packages url
+                            sh "mvn --batch-mode versions:set-property -Dproperty=plugin.unreleased.repository.url -DnewVersion=\"${config.maven_packages_url}\" -DgenerateBackupPoms=false"
```

Comment:
I think you want `maven_packages_url` instead of `config.maven_packages_url`. The former appends the build number and `/maven` suffix.

Reply:
ah thanks, though I'm currently running into build issues even before that step... (that's why it's still a draft haha, but posting it here maybe you have some insight)
```
[2020-05-28T20:22:40.615Z] [ERROR]   The project io.confluent.ksql:ksqldb-parent:6.0.0-SNAPSHOT (/home/jenkins/workspace/confluentinc-pr_ksql_PR-5500/ksql-db/pom.xml) has 1 error
[2020-05-28T20:22:40.615Z] [ERROR]     Non-resolvable parent POM for io.confluent.ksql:ksqldb-parent:6.0.0-SNAPSHOT: Could not transfer artifact io.confluent:common:pom:6.0.0-SNAPSHOT from/to confluent-snapshots (${plugin.unreleased.repository.url}): Cannot access ${plugin.unreleased.repository.url} with type default using the available connector factories: BasicRepositoryConnectorFactory and 'parent.relativePath' points at wrong local POM @ line 22, column 13: Cannot access ${plugin.unreleased.repository.url} using the registered transporter factories: WagonTransporterFactory: Unsupported transport protocol -> [Help 2]
```